include/libxnvme_spec.h

```diff
@@ -1345,7 +1345,20 @@ struct xnvme_spec_cmd_nvm {
 	uint32_t fua    : 1; ///< FUA: Force unit access
 	uint32_t lr     : 1; ///< LR: Limited retry
 
-	uint32_t cdw13_15[3]; ///< Command dword 13 to 15
+	/* cdw 13 */
+	uint32_t af     : 4; ///< Access Frequency
+	uint32_t al     : 2; ///< Access Latency
+	uint32_t sr     : 1; ///< Sequential Reques
```

Comment:
typo: ``Sequential Reques`` -> ``Sequential Request``.
It might collide with something elsewhere, but how about wrapping these DSM fields in an anon.struct. for access with:
```
dsm.incom
```
Plus a "raw" access for all 8 bits?

Reply:
I wrapped for the complete cdw13, like this 

        union {
                struct {
                        uint32_t af    : 4; ///< Access Frequency
                        uint32_t al    : 2; ///< Access Latency
                        uint32_t sr    : 1; ///< Sequential Request
                        uint32_t incom : 1; ///< Incompressible
                        uint32_t rsvd3 : 8;
                        uint32_t dspec : 16; ///< Directive Specific
                };
                uint32_t val;
        } cdw13;

Hope this is ok.

Reply:
Even better.